invenio_rdm_records/services/services.py

```diff
@@ -293,31 +291,54 @@ def get_client(self, client_name):
 
         return client_class(name=client_name)
 
-    def get_provider(self, scheme, client_name=None):
+    def _get_managed_provider(self, providers_dict):
```

Comment:
minor: in both cases we are assuming a preference for the first configure as such (e.g. the first managed)... We chould state this in the docs.

Reply:
yes, you are right. Here we should have max 1 managed and max 1 unmanaged.
Maybe the config should enforce this, or we document it when we document when to override.
We cannot have 2 managed for sure...